src/site/content/en/handbook/audience/index.md

```diff
@@ -55,17 +56,15 @@ Jaimie first took a web development class at their local community college and i
 
 ## Writing to the audience
-All posts and codelabs should be targeted to one of our personas (or more, if their needs overlap).
-
-That said, we want our content to address as much of our audience as we can. How do you do that in a given piece, especially when the needs of different readers compete? One way to address multiple audience segments is to provide multiple paths for a given topic. For example, you might write:
+When you're writing a post or codelab, keep the web.dev personas in mind (or more, if their needs overlap). How do you do that in a given piece, especially when the needs of different readers compete? One way to address multiple audience segments is to provide multiple paths for a given topic. For example, you might write:
 * A post using vanilla JavaScript, with associated codelabs using common frameworks to solve the same problem
 * A post about optimizing images in general, with associated codelabs showing how to integrate image optimization into different build processes
-* A post about how to address an issue most web developers will encounter, with sections at the end focused on solving the problem as it appears in particular business verticals
+* A post about how to address an issue most web developers encounter, with sections at the end focused on solving the problem as it appears in particular business verticals
 
-Be mindful of your [voice](/handbook/voice). You want readers to feel they can trust the guidance you're giving. They should feel neither talked down to nor overwhelmed. Check out "[Instruction for adult learners](/handbook/effective-instruction#instruction-for-adult-learners)" in the "[Writing effective instruction](/handbook/effective-instruction)" post.
+Be mindful of your [voice](/handbook/voice). You want readers to feel they can trust the guidance you're giving. They should feel neither talked down to nor overwhelmed. Check out [Instruction for adult learners](/handbook/effective-instruction#instruction-for-adult-learners) in the [Writing effective instruction](/handbook/effective-instruction) post.
 
 ### Value propositions
-A value proposition is a promise to readers that a piece of content will be useful—that is, provide value—to them. Create a value proposition for everything you write by crafting titles, subtitles, and descriptions that tell your target audience why your post or codelab is worth reading. Some common strategies:
-* Say what readers will learn or understand by reading the piece
-* Say what a piece will let readers do
-* Say how a technique or tool will improve the experience of readers' users
+A value proposition is a promise to readers that a piece of content will be useful and provide value to them. Create a value proposition for everything you write by crafting titles, subtitles, and descriptions that tell your target audience why your post or codelab is worth reading. Here are some common strategies:
```

Comment:
```suggestion
A value proposition is a promise to readers that a piece of content will be useful to them. Create a value proposition for everything you write by crafting titles, subtitles, and descriptions that tell your target audience why your post or codelab is worth reading. Here are some common strategies:
```
I was thinking that "be useful" and "provide value" are equivalent here. The goal of the aside was to connect the phrasing of the key term to its definition. If you don't think that's needed, I'd just cut it altogether to avoid redundancy.